### PR TITLE
Release version 0.16.0

### PR DESCRIPTION
**Pull request by tijmenb**
Add `Smartdown::Api::Flow#content_id` (https://github.com/alphagov/smartdown/pull/147)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 0.16.1
```
Comment (KushalP) [inline CHANGELOG.md]: Why are you incrementing both the minor version and the patch version here? I'm wondering if this should be `0.16.0` ?

Comment (tijmenb) [inline CHANGELOG.md]: Heh, yeah. Thanks.

**Comment by alext** [comment]
For gems, it's better to pin to a minor version (i.e. 2.2) rather than a specific version, that way it'll always be build against the latest 2.2.x. (We create rbenv aliases in all environments so that 2.2 points at the latest 2.2.x)

**Comment by tijmenb** [comment]
@alext Thanks, didn't know that. Updated.

**Pull request merged**